cmd/protoc-gen-docs/htmlGenerator.go

```diff
@@ -1111,6 +1113,27 @@ func getFieldBehavior(options *descriptor.FieldOptions) []annotations.FieldBehav
 	return s
 }
 
+func getRequiredFromComments(loc protomodel.LocationDescriptor) bool {
+	com := loc.GetLeadingComments()
+	if com == "" {
+		com = loc.GetTrailingComments()
+		if com == "" {
+			return false
+		}
+	}
+
+	if strings.Contains(com, "Required") {
+		return true
+	}
+
+	if strings.Contains(com, "Optional") {
+		return true
```

Comment:
It seems like Optional and default being true, this will always return true, and they should be false.